realm/realm-jni/src/io_realm_internal_tableview.cpp

```diff
@@ -1015,3 +1015,17 @@ JNIEXPORT jlong JNICALL Java_io_realm_internal_TableView_nativeSync(
     } CATCH_STD()
     return 0;
 }
+
+JNIEXPORT jlong JNICALL Java_io_realm_internal_TableView_nativeFindBySourceNdx
+        (JNIEnv *env, jobject, jlong nativeViewPtr, jlong rowIndex)
+{
+    TR_ENTER_PTR(nativeViewPtr);
+    try {
+        if (!VIEW_VALID_AND_IN_SYNC(env, nativeViewPtr) || !ROW_INDEX_VALID(env, TV(nativeViewPtr), rowIndex))
```

Comment:
Then it can be `ROW_INDEX_VALID(env, &(TV(nativeViewPtr)->getParent()), rowIndex))`